.gitignore

```diff
@@ -97,6 +97,7 @@ dlldata.c
 project.lock.json
 artifacts/
 bin/
+artifactsSandBox/
```

Comment:
Can we call this something different? 
"artifacts" was removed - so "artifactsSandBox" doesn't seem desirable.

You move _everything_ into artifactsSandBox and then work from artifactsSandBox, correct?

How about "WorkingCopy" or something like that...

Reply:
Changed to bin/WorkingCopy